components/Sidebar.tsx

```diff
@@ -15,9 +16,10 @@ import ScrollToTop from "./ScrollToTop";
 import { SectionTitle } from "./SectionTitle";
 import { SponsorSection } from "./Sponsor/SponsorSection";
 
+export const GithubLogo = () => <FontAwesomeIcon icon={faGithub} className="mr-2" />;
```

Comment:
We could just use the `FontAwesome` component directly to avoid having too many. If we keep this, then maybe a separate file would be better than having this defined in the Sidebar.